docs/the-new-architecture/cxx-cxxturbomodules.md

```diff
@@ -0,0 +1,522 @@
+---
+id: cxx-cxxturbomodules
+title: C++ Turbo Native Modules
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/TabsConstants';
+import NewArchitectureWarning from '../\_markdown-new-architecture-warning.mdx';
+
+This guide shows you how to implement a Turbo Native Module using C++ only, a way to share the same implementation with any supported platform (Android, iOS, macOS or Windows).
```

Comment:
As per all the others pages in the New Architecture guides, we should add the yellow experimental banner.
```
import NewArchitectureWarning from '../\_markdown-new-architecture-warning.mdx';

<NewArchitectureWarning/>
```